src/main/kotlin/com/dansplugins/factionsystem/MedievalFactions.kt

```diff
@@ -238,6 +232,64 @@ class MedievalFactions : JavaPlugin() {
         )
         setupRpkLockService()
 
+        val metrics = Metrics(this, 8929)
+        metrics.addCustomChart(
+            SimplePie("language_used") {
+                config.getString("language")
+            }
+        )
+        metrics.addCustomChart(
+            SimplePie("database_dialect") {
+                config.getString("database.dialect")
+            }
+        )
+        metrics.addCustomChart(
+            SimplePie("average_claims") {
+                factionService.factions
+                    .map {
+                        claimService.getClaims(it.id).size
+                    }
+                    .average().toInt().toString()
```

Comment:
Does this need .toInt()? If it needs to be an int perhaps .roundToInt()?

Reply:
Changed this to .roundToInt()!